diff --git a/mkfs/rootdir.c b/mkfs/rootdir.c
index 8d626f10..a452c005 100644
--- a/mkfs/rootdir.c
+++ b/mkfs/rootdir.c
@@ -278,16 +278,16 @@ static void stat_to_inode_item(struct btrfs_inode_item *dst, const struct stat *
 		btrfs_set_stack_inode_size(dst, st->st_size);
 	btrfs_set_stack_inode_nbytes(dst, 0);
 	btrfs_set_stack_inode_block_group(dst, 0);
-	btrfs_set_stack_inode_uid(dst, st->st_uid);
-	btrfs_set_stack_inode_gid(dst, st->st_gid);
+	btrfs_set_stack_inode_uid(dst, 0);
+	btrfs_set_stack_inode_gid(dst, 0);
 	btrfs_set_stack_inode_mode(dst, st->st_mode);
 	btrfs_set_stack_inode_rdev(dst, 0);
 	btrfs_set_stack_inode_flags(dst, 0);
-	btrfs_set_stack_timespec_sec(&dst->atime, st->st_atime);
+	btrfs_set_stack_timespec_sec(&dst->atime, 1);
 	btrfs_set_stack_timespec_nsec(&dst->atime, 0);
-	btrfs_set_stack_timespec_sec(&dst->ctime, st->st_ctime);
+	btrfs_set_stack_timespec_sec(&dst->ctime, 1);
 	btrfs_set_stack_timespec_nsec(&dst->ctime, 0);
-	btrfs_set_stack_timespec_sec(&dst->mtime, st->st_mtime);
+	btrfs_set_stack_timespec_sec(&dst->mtime, 1);
 	btrfs_set_stack_timespec_nsec(&dst->mtime, 0);
 	btrfs_set_stack_timespec_sec(&dst->otime, 0);
 	btrfs_set_stack_timespec_nsec(&dst->otime, 0);
